src/App.tsx

```diff
@@ -251,10 +251,10 @@ const Main = ({ children, ...props }: ViewProps) => {
 
   useEffect(() => {
     const handleAppStateChange = (nextAppState: AppStateStatus) => {
-      if (nextAppState === 'background' && !isCameraOpen) {
+      if (nextAppState === 'background' && walletMnemonic && !isCameraOpen) {
```

Comment:
We only trigger the "appBecameInactive" when we actually have a mnemonic (meaning, a wallet). Otherwise, it means we don't have one yet, so we are in the wallet creation flow.